config.json

```diff
@@ -1086,7 +1102,13 @@
         "name": "Beer Song",
         "uuid": "6573f168-d8fc-4ccf-a864-1a61f432fae1",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "arrays",
+          "conditionals",
+          "recursion",
+          "template-strings",
+          "while-loops"
```

Comment:
I would recommend `for-loops` here instead of recursion and while, see example solution in the repo.